game-core/src/test/java/games/strategy/triplea/delegate/battle/MustFightBattleExecutablesTest.java

```diff
@@ -1603,68 +1602,4 @@ void firstStrikeDefenderReturnFireAttNoDestroyerDefNoDestroyerWW2v2FalseSneakAtt
         MustFightBattle.ReturnFire.ALL,
         false);
   }
-
-  @Test
-  @DisplayName("Verify attacking canNotBeTargetedByAll can submerge if all is air")
```

Comment:
These two tests are now being done in `SubmergeSubsVsOnlyAirStepTest#testSubmerging`.  I considered leaving these tests in till after this is merged just to show that the logic is still correct.  Should I put them back in or is it ok to delete them now?